app/scripts/schemas/schemas-service.js

```diff
@@ -334,6 +334,8 @@
                                     break;
                                 case 'enumSource': // Companion to 'watch', but not used here
                                     break;
+                                case 'minLength':  // String fields need a 'minLength' field to be 'required',
+                                    break;         // but the user shouldn't be able to edit it directly.
```

Comment:
This change helps prevent the `minLength` field from showing up in the schema editor. (We use `minLength` to help enforce that string fields be `required`, but in newer versions of JSONEditor it interprets this field as an editable field and tries to show it to the end user.)